File: src/java/eu/sig/training/ch06/simpledigitalcamera/DigitalCamera.java
package eu.sig.training.ch06.simpledigitalcamera;

import java.awt.Image;

// tag::DigitalCamera[]
public class DigitalCamera implements SimpleDigitalCamera {
    // ...
    // end::DigitalCamera[]
    public Image takeSnapshot() {
        return null;
    }

    public void flashLightOn() {}

    public void flashLightOff() {}
    // tag::DigitalCamera[]
}
// end::DigitalCamera[]

File: src/csharp/eu/sig/training/ch06/simpledigitalcamera/DigitalCamera.cs
using System.Drawing;

namespace eu.sig.training.ch06.simpledigitalcamera
{

    // tag::DigitalCamera[]
    public class DigitalCamera : ISimpleDigitalCamera
    {
        // ...
        // end::DigitalCamera[]
        public Image TakeSnapshot()
        {
            return null;
        }

        public void FlashLightOn()
        {
        }

        public void FlashLightOff()
        {
        }
        // tag::DigitalCamera[]
    }
    // end::DigitalCamera[]
}
